moses/moses/representation/instance_scorer.cc

```diff
@@ -76,7 +76,7 @@ composite_score atomese_based_scorer::operator()(const instance &inst) const
 	}
 
 	combo_tree tr = _rep.get_candidate(inst, _reduce);
-	Handle handle = atomese_combo(tr);
+	Handle handle = _comboToAtomeseConverter(tr);
```

Comment:
I wouldn't recommend having such long object name. Besides the opencog dev standard prefers snake case over caml cases (except for class names). So maybe something like `_combo_to_atomese`, or even simply `_to_atomese`.